### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by karenkrieger**
Creating the Contributing file

#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.
- [X] No, and I am not going to.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+- New content (endpoints, descriptions or fields from scratch)
+- Improvement (make an endpoint's title or description even better)
+- Spelling and grammar accuracy (self-explanatory)
+
+### Style Guidelines
+We created a [OpenAPI checklist](https://github.com/vtex/openapi-schemas/blob/master/OpenAPI%20Checklist.md) that contains all fields that we consider required for our API documentation. Please read the checklist before contributing.
+
+### Review
+You must add the [@technical-writers](https://github.com/orgs/vtex-apps/teams/technical-writers/members) to review your contribution. By adding the team, we will indicate the right person to review your PR.
```
Comment (PedroAntunesCosta) [inline CONTRIBUTING.md:17]:
```suggestion
You must add the [@technical-writers](https://github.com/orgs/vtex-apps/teams/technical-writers/members) as your contribution's reviewers. By adding the team, we will indicate the right person to review your PR.
```

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
Só sugeri mudança pequena.

**Review by brunoamui — CHANGES_REQUESTED** [review]
Senti falta de um link para esse arquivo no readme do repositório.

**Review by brunoamui — APPROVED** [review]
LGTM

**Pull request merged**